pkg/cmd/webhook/update/update.go

```diff
@@ -88,6 +89,7 @@ func NewCmdWebHookVerify() (*cobra.Command, *Options) {
 	cmd.Flags().StringVarP(&o.HMAC, "hmac", "", "", "Don't use the HMAC token from the cluster, use the provided token")
 	cmd.Flags().StringVarP(&o.Endpoint, "endpoint", "", "", "Don't use the endpoint from the cluster, use the provided endpoint")
 	cmd.Flags().BoolVarP(&o.WarnOnFail, "warn-on-fail", "", false, "If enabled lets just log a warning that we could not update the webhook")
+	cmd.Flags().StringVarP(&o.EnforceNamespace, "enforce-namespace", "", "", "Disable namespace auto detection, instead use specified namespace")
```

Comment:
can be confusing, how about &DevNamespace? 
`StringVarP(&o.DevNamespace, "dev-ns",`
This argument is ultimately setting/getting the dev namespace. wdyt?

Reply:
Yup. The autodetection mechanism is taking the namespace from the `team` label of the namespace as I remember. It seems also that the concept of `team` was in JX v2.

Reply:
Will change to dev-ns.